ajax/libs/jquery-date-range-picker/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery-date-range-picker",
+  "description": "jQuery Date Range Picker is a jQuery plugin that allows user to select a date range.",
+  "version": "0.5.1",
+  "license": "MIT",
+  "keywords": [
+    "jquery",
+    "date",
+    "daterange",
+    "date",
+    "picker",
+    "range"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/longbill/jquery-date-range-picker.git"
+  },
+  "homepage": "https://github.com/longbill/jquery-date-range-picker",
```

Comment:
We don't use github page as the homepage. 
If you don't have the homepage you can remove this field.